packages/api/src/router/like.ts

```diff
@@ -5,6 +5,22 @@ import { z } from "zod";
 import { createTRPCRouter, protectedProcedure } from "../trpc";
 
 export const likeRouter = createTRPCRouter({
+  getLikesForDiscussion: protectedProcedure
+    .input(
+      z.object({
+        postId: z.number().nonnegative(),
+      }),
+    )
+    .query(async ({ input, ctx }) => {
+      const likes = await ctx.db.likes.findMany({
+        where: { postId: input.postId },
+      });
+
+      const isLikedByUser = likes.some((like) => like.userId === ctx.userId);
```

Comment:
I feel like this method can be its' own endpoint but this is up to you. Try to keep the functionality of an endpoint to 1 thing